frontend/form-builder/src/app/App.module.css

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
Kan vurdere å lage en reset.css fil for å håndtere overstyring av browser spesifikke ting. :)

Reply:
Takk! 😊 Har lagd en reset.css nå. Er det innafor å fjerne list-stylingen til `<ul>` i den?

```
* {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
}

ul {
    list-style: none;
}
```

Reply:
Det er litt opp til prosjekt til prosjekt, men tenker det er innafor ja :)